lib/frameworks/jasmine2.js

```diff
@@ -86,6 +104,16 @@ exports.run = function(runner, specs) {
     return true;
   };
 
+  function isSuiteMatched(suite) {
+    if (!suite) {
```

Comment:
Should this `if` be here?  Shouldn't we just skip all suite checks if `suite == undefined`?